### PR TITLE
Add more logs to new Stripe plugin

### PR DESCRIPTION
**Pull request by korycins**
I want to merge this change because it adds more logs to stripe logic

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-3524 Improve logging for Stripe Plugin](https://app.clickup.com/t/2549495/SALEOR-3524)

File: saleor/payment/gateways/stripe/plugin.py [hunk 1]
```diff
@@ -456,7 +464,7 @@ def validate_plugin_configuration(cls, plugin_configuration: "PluginConfiguratio
                 raise ValidationError(
                     {
                         field: ValidationError(
-                            "The parameter is required.",
+                            "The parameter is required",
```
Comment (maarcingebala) [inline saleor/payment/gateways/stripe/plugin.py:467]: Why removing the period? I think most of our messages have it 🤔  🚔

File: saleor/payment/gateways/stripe/stripe_api.py [hunk 2]
```diff
@@ -198,8 +244,15 @@ def cancel_payment_intent(
...
     except StripeError as error:
         logger.warning(
-            "Unable to cancel a payment intent (%s), error", payment_intent_id
+            "Unable to cancel a payment intent",
+            extra={
+                "error_message": error.user_message,
+                "http_status": error.http_status,
+                "code": error.code,
+                "payment_intent": payment_intent_id,
+            },
```
Comment (IKarbowiak) [inline saleor/payment/gateways/stripe/stripe_api.py:253]: As `extra` data are repeated few times, maybe we should create some function for preparing this?

**Review by fowczarek — APPROVED** [review]

File: saleor/payment/gateways/stripe/stripe_api.py (hunk 2 above)
Comment (korycins) [reply saleor/payment/gateways/stripe/stripe_api.py:253]: Do you mean a function that will take params and create from it a dictionary? extra is repeated many times, but most of the calls have different arguments.

File: saleor/payment/gateways/stripe/plugin.py (hunk 1 above)
Comment (korycins) [reply saleor/payment/gateways/stripe/plugin.py:467]: Will bring it back as it is something that is returned by API.

**Review by tomaszszymanski129 — APPROVED** [review]

**Pull request merged**